addons/l10n_ch/__init__.py

```diff
@@ -14,9 +14,7 @@ def init_settings(env):
     '''If the company is localized in Switzerland, activate the cash rounding by default.
     '''
     # The cash rounding is activated by default only if the company is localized in Switzerland.
-    for company in env['res.company'].search([]):
-        if company.country_id != env.ref('base.ch'):
-            continue
+    for company in env['res.company'].search([('partner_id.country_id', '=', 'ch')]):
```

Comment:
actually, are you sure this is working?
Would it not be better to have smth like this:
```
ch_country = env.ref('base.ch')
for company in env['res.company'].search([('partner_id.country_id', '=', ch_country)]):
```

Reply:
Yep it's working, I have test your solution and it's working too but with a little modification:
`
ch_country = env.ref('base.ch')
for company in env['res.company'].search([('partner_id.country_id', '=', ch_country.id)]):
`
We need to put **.id** after the `ch_country` but it's work. I will push the modification now ! 
Thanks